### PR TITLE
(MAINT) fix pathing to inventory.yaml and README instructions

### PR DESCRIPTION
**Pull request by ThoughtCrhyme**

### CODE REVIEW

File: plans/agents.pp [hunk 1]
```diff
@@ -4,13 +4,15 @@
...
   $agents = get_targets('*').filter |$n| { $n.vars['role'] =~ /agent/ }
+  $windows_agents = get_targets('*').filter |$n| { $n.vars['role'] == "agent_windows" }
 
 
   # install agents
   run_task('puppet_agent::install', $agents)
   # set the server 
   $server_string = $server[0].name
   run_task('puppet_conf', $agents, action => 'set', section => 'main', setting => 'server', value => "${server_string}")
+  run_command("powershell.exe -NoProfile -Nologo -Command 'Remove-Item -Path /ProgramData/PuppetLabs/puppet/etc/ssl -Force -Recurse'", $windows_agents)
```
Comment (ThoughtCrhyme) [inline plans/agents.pp:15]: I have to be honest, I don't really know why nuking the ssl directory works, but it does :/

**Review by tphoney — CHANGES_REQUESTED** [review]
we should add the teardown task to the readme

File: plans/agents.pp (hunk 1 above)
Comment (tphoney) [reply plans/agents.pp:15]: yeah that's disappointing :( likely to do with how puppet_agent is working on windows.

File: plans/tester.pp [hunk 2]
```diff
@@ -3,10 +3,16 @@
   # get pe_server ?
   $server = get_targets('*').filter |$n| { $n.vars['role'] == 'pe' }
   $agent_linux = get_targets('*').filter |$n| { $n.vars['role'] =~ /agent_linux/ }
-
+  $agent_windows = get_targets('*').filter |$n| { $n.vars['role'] =~ /agent_windows/ }
```
Comment (tphoney) [inline plans/tester.pp]: can you make the regex  /agent/ then we can just run against all agents, rather than differentiating

Comment (ThoughtCrhyme) [reply plans/tester.pp]: done

**Review by tphoney — APPROVED** [review]

**Pull request merged**